permissions-gallery/build.gradle.kts

```diff
@@ -1,19 +1,44 @@
+import io.gitlab.arturbosch.detekt.CONFIGURATION_DETEKT_PLUGINS
+
 /*
  * Copyright 2024 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
  */
 
 plugins {
-    id("dev.icerock.moko.gradle.multiplatform.mobile")
+    id("org.jetbrains.kotlin.multiplatform")
+    id("com.android.library")
     id("dev.icerock.moko.gradle.publication")
     id("dev.icerock.moko.gradle.stub.javadoc")
-    id("dev.icerock.moko.gradle.detekt")
+    alias(libs.plugins.detekt)
 }
 
 android {
     namespace = "dev.icerock.moko.permissions.gallery"
+    compileSdk = 36
```

Comment:
this provided from moko gradle setup.
you should rollback moko convention plugins.